starport/services/network/reward.go

```diff
@@ -0,0 +1,61 @@
+package network
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	rewardtypes "github.com/tendermint/spn/x/reward/types"
+
+	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/services/network/networktypes"
+)
+
+// SetReward set a chain reward
+func (n Network) SetReward(launchID, lastRewardHeight uint64, coins sdk.Coins) error {
+	n.ev.Send(events.New(
+		events.StatusOngoing,
+		fmt.Sprintf("Setting reward %s to the chain %d at height %d",
+			coins.String(),
+			launchID,
+			lastRewardHeight,
+		),
+	))
+
+	msg := rewardtypes.NewMsgSetRewards(
+		n.account.Address(networktypes.SPN),
+		launchID,
+		lastRewardHeight,
+		coins,
+	)
+	res, err := n.cosmos.BroadcastTx(n.account.Name, msg)
+	if err != nil {
+		return err
+	}
+
+	var setRewardRes rewardtypes.MsgSetRewardsResponse
+	if err := res.Decode(&setRewardRes); err != nil {
+		return err
+	}
+
+	doneMsg := fmt.Sprintf("%s will be distributed to validators at height %d", coins.String(), lastRewardHeight)
+	if setRewardRes.PreviousCoins.Empty() {
+		doneMsg = fmt.Sprintf("New reward pool was created.\n%s", doneMsg)
+	} else {
+		doneMsg = fmt.Sprintf("Previous reward pool %s at height %d was overwritten.\n%s",
```

Comment:
Newlines in events will probably look bad when they're printed because they won't have bullets. What do you think emitting another event for each line here?

Let's take an example, I'm setting a reward for the first time, and this what I get printed:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/6023117/155424817-a7a9b47e-30b2-45d2-b2fb-50eb95f97014.png">

It would have be better if we have seen something like this instead:
```
✔ New reward pool was created.
ℹ 10uspn will be distributed to validators at height 22
ℹ The chain 1 is now an incentivized testnet
```

Btw as you see, I used ℹ char for the other two events but by default, events only use ✔.

So, I also propose to add ability to customize this with an API like:
```go
events.New(status Status, description string, options ...Option)

events.New(events.StatusDone, "text...", events.BulletInfo())
// By default it would use ✔.
```

Btw, can we make sure that we re not using emoji version of ℹ since we're moving from emojis?

Can we also give it a yellow color?